explore/Target_therapy.csv

```diff
@@ -0,0 +1,47 @@
+Drug,Entrez_gene_id,Gene_symbol

```

Comment:
can we remove capitalization of first letter? Specifically, `entrez_gene_id` is the column name we use in `cognoma/genes` and `cognoma/cancer-data`.